### PR TITLE
Routine update of hledger to the latest release

### PR DESCRIPTION
**Pull request by simonmichael**
Update hledger to latest 1.15.1, drop obsolete hledger-api tool.

### CODE REVIEW

File: Formula/hledger.rb [hunk 1]
```diff
@@ -5,44 +5,32 @@ class Hledger < Formula
 
   desc "Command-line accounting tool"
   homepage "https://hledger.org/"
-  url "https://hackage.haskell.org/package/hledger-1.14.2/hledger-1.14.2.tar.gz"
-  sha256 "849a6e0683192ec504da9a631ddfc82e04973583f4a028fd39b8cdac2efe29ea"
+  url "https://hackage.haskell.org/package/hledger-1.15.1/hledger-1.15.1.tar.gz"
```
Comment (jonchang) [inline Formula/hledger.rb]: please add the correct `sha256` for this file. Try: `curl -L https://hackage.haskell.org/package/hledger-1.14.2/hledger-1.14.2.tar.gz | shasum -a 256`

File: Formula/hledger.rb [hunk 2]
```diff
@@ -5,44 +5,32 @@ class Hledger < Formula
...
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "f8f717413411bf98616d3df20f31583928b8b93e937080632e08677512cd9d5b" => :mojave
-    sha256 "5150a38fc8378410ca9b7b9d2fca755ccc7fe0eef6805fc07650fd836460a0bb" => :high_sierra
-    sha256 "60422bb49e6e62f26780cec246007cfa38dc8fcb33c6718591e7f7e72fb44c2a" => :sierra
```
Comment (jonchang) [inline Formula/hledger.rb]: Do not modify `bottle` blocks for version bumps.

File: Formula/hledger.rb [hunk 3]
```diff
@@ -5,44 +5,32 @@ class Hledger < Formula
...
   end
 
   depends_on "cabal-install" => :build
   depends_on "ghc" => :build
   uses_from_macos "ncurses"
 
   resource "hledger_web" do
-    url "https://hackage.haskell.org/package/hledger-web-1.14.1/hledger-web-1.14.1.tar.gz"
-    sha256 "a1eacde5b9d531df0875b65c8239e8351749610e1e6e46c847dd02594fb6a970"
```
Comment (jonchang) [inline Formula/hledger.rb]: Please add the correct `sha256`

File: Formula/hledger.rb [hunk 4]
```diff
@@ -5,44 +5,32 @@ class Hledger < Formula
...
 
   resource "hledger_ui" do
-    url "https://hackage.haskell.org/package/hledger-ui-1.14.2/hledger-ui-1.14.2.tar.gz"
-    sha256 "9951a8665c7a182d8008c92565272a6c4a8e12d363df4b169fa09dddffee112e"
-  end
-
-  resource "hledger_api" do
-    url "https://hackage.haskell.org/package/hledger-api-1.14/hledger-api-1.14.tar.gz"
-    sha256 "ad7a714201cf912a6c756e40a25116e2352b86a81b048599c15f403b2a65f7a3"
+    url "https://hackage.haskell.org/package/hledger-ui-1.15/hledger-ui-1.15.tar.gz"
```
Comment (jonchang) [inline Formula/hledger.rb]: Please add the correct `sha256`

**Review by jonchang — CHANGES_REQUESTED** [review]

**Comment by simonmichael** [comment]
@jonchang, thanks for the tips - new commit pushed.

**Comment by simonmichael** [comment]
@jonchang: ready for your merge. Thanks!

**Review by jonchang — APPROVED** [review]

**Comment by jonchang** [comment]
Thanks for making these changes @simonmichael! Please amend the commit message to [conform to our commit message style](https://docs.brew.sh/Formula-Cookbook#commit), then rebase on master, force-push, and ping me so we can get this update merged.

**Comment by simonmichael** [comment]
Thanks @jonchang - done. I appreciate the almost there tag. :-)

**Comment by jonchang** [comment]
Please update the commit message to `hledger 1.15.1`

**Comment by simonmichael** [comment]
Oops. Done, @jonchang.

**Comment by fxcoudert** [comment]
Thanks @simonmichael for your contribution to Homebrew!

**Pull request closed without merging**